package.json

```diff
@@ -69,6 +69,7 @@
     "file-loader": "4.3.0",
     "fs-extra": "^8.1.0",
     "graphql": "^14.6.0",
+    "hds-react": "^0.5.4",
```

Comment:
So, you decided to use the react module despite the fact we agreed not to?

Reply:
Check your notes. We specifically decided to use it for icons.